keanu-python/keanu/vertex/__init__.py

```diff
@@ -1,7 +1,7 @@
 ## This is a generated file. DO NOT EDIT.
 
 # This allows us to make these vertices visible externally so they can be picked up for our python docs
-__all__ = ["BooleanIf", "BooleanProxy", "CastToBoolean", "ConstantBoolean", "NumericalEquals", "AndBinary", "OrBinary", "Equals", "GreaterThanOrEqual", "GreaterThan", "LessThanOrEqual", "LessThan", "NotEquals", "BooleanConcatenation", "BooleanReshape", "BooleanSlice", "BooleanTake", "Not", "Bernoulli", "CastToDouble", "ConstantDouble", "DoubleIf", "DoubleProxy", "Addition", "ArcTan2", "Difference", "Division", "MatrixMultiplication", "Max", "Min", "Multiplication", "Power", "Concatenation", "Abs", "ArcCos", "ArcSin", "ArcTan", "Ceil", "Cos", "Exp", "Floor", "LogGamma", "Log", "MatrixDeterminant", "MatrixInverse", "Reshape", "Round", "Sigmoid", "Sin", "Slice", "Sum", "Take", "Tan", "Beta", "Cauchy", "ChiSquared", "Dirichlet", "Exponential", "Gamma", "Gaussian", "HalfCauchy", "HalfGaussian", "InverseGamma", "KDE", "Laplace", "LogNormal", "Logistic", "MultivariateGaussian", "Pareto", "SmoothUniform", "StudentT", "Triangular", "Uniform", "Print", "CastToInteger", "ConstantInteger", "IntegerIf", "IntegerProxy", "IntegerAddition", "IntegerDifference", "IntegerDivision", "IntegerMax", "IntegerMin", "IntegerMultiplication", "IntegerPower", "IntegerConcatenation", "IntegerAbs", "IntegerReshape", "IntegerSlice", "IntegerSum", "IntegerTake", "Binomial", "Multinomial", "Poisson", "UniformInt", "Assert"]
+__all__ = ["BooleanIf", "BooleanProxy", "CastToBoolean", "ConstantBoolean", "NumericalEquals", "AndBinary", "OrBinary", "Equals", "GreaterThanOrEqual", "GreaterThan", "LessThanOrEqual", "LessThan", "NotEquals", "BooleanConcatenation", "BooleanReshape", "BooleanSlice", "BooleanTake", "Not", "Bernoulli", "CastToDouble", "ConstantDouble", "DoubleIf", "DoubleProxy", "Addition", "ArcTan2", "Difference", "Division", "MatrixMultiplication", "Max", "Min", "Multiplication", "Power", "Concatenation", "Abs", "ArcCos", "ArcSin", "ArcTan", "Ceil", "Cos", "Exp", "Floor", "LogGamma", "Log", "MatrixDeterminant", "MatrixInverse", "Permute", "Reshape", "Round", "Sigmoid", "Sin", "Slice", "Sum", "Take", "Tan", "Beta", "Cauchy", "ChiSquared", "Dirichlet", "Exponential", "Gamma", "Gaussian", "HalfCauchy", "HalfGaussian", "InverseGamma", "KDE", "Laplace", "LogNormal", "Logistic", "MultivariateGaussian", "Pareto", "SmoothUniform", "StudentT", "Triangular", "Uniform", "Print", "CastToInteger", "ConstantInteger", "IntegerIf", "IntegerProxy", "IntegerAddition", "IntegerDifference", "IntegerDivision", "IntegerMax", "IntegerMin", "IntegerMultiplication", "IntegerPower", "IntegerConcatenation", "IntegerAbs", "IntegerReshape", "IntegerSlice", "IntegerSum", "IntegerTake", "Binomial", "Multinomial", "Poisson", "UniformInt", "Assert"]
```

Comment:
Nice, looks like the permute was missing.

Reply:
👍